app.js

```diff
@@ -0,0 +1,14 @@
+const express = require ('express');
+const app = express();
+const {fetchTopic} = require("./controllers/ncnews_controllers")
+
+
+app.get('/api/topics', fetchTopic)
+app.get('/api/topicss')
+
```

Comment:
why do you have an endpoint for `/api/topicss`?

Reply:
End point omitted and test block for handling 404 error added to the first end point on line 6